Linguini.Shared/Types/Bundle/FluentNumber.cs

```diff
@@ -45,7 +45,7 @@ public string AsString()
 
         public static FluentNumber FromString(ReadOnlySpan<char> input)
         {
-            var parsed = Double.Parse(input);
+            var parsed = Double.Parse(input, NumberStyles.Float, CultureInfo.InvariantCulture);
```

Comment:
Shouldn't `NumberStyles.Float` be `NumberStyles.Float | NumberStyles.AllowThousands`? I'd like to not change parsing rules between `Double.Parse` and `FromString`

Reply:
Updated.